index.html

```diff
@@ -218,7 +218,7 @@ <h2>Tonic</h2>
       </section>
 
       <section id="about">
-        <div>
+          <div class="desktop-about textside">
           <h2>About<br />Myself</h2>
           <p>
```

Comment:
- [ ] It is required to stick to the figma design as close as possible. Your design for about myself section is a bit different from what we have on the Figma design. You need to stick to the design as much as possible. The involves font-size, font-family, color, background color, margin, padding among others. Kindly compare your design and work it alongside the Figma design.

### Your Design
![your-about-mobile](https://user-images.githubusercontent.com/26597432/220654344-62f6cbbf-3275-4df7-8a87-bec45901532f.png)

### Figma Design
![temp-about-mobile](https://user-images.githubusercontent.com/26597432/220654400-94c7911c-4c80-4573-bfe3-f03768cda062.png)

Reply:
fixed